src/libs/OptionsListUtils.js

```diff
@@ -159,17 +161,20 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     searchValue = '',
     showChatPreviewLine = false,
     showReportsWithNoComments = false,
+    hideReadReports = false,
+    sortByAlphaAsc = false,
 }) {
     let recentReportOptions = [];
     const pinnedReportOptions = [];
     const personalDetailsOptions = [];
 
     const reportMapForLogins = {};
-    const orderedReports = lodashOrderBy(reports, [
-        sortByLastMessageTimestamp
-            ? 'lastMessageTimestamp'
-            : 'lastVisitedTimestamp',
-    ], ['desc']);
+    let iteratees = sortByLastMessageTimestamp
+        ? ['lastMessageTimestamp']
+        : ['lastVisitedTimestamp'];
+    if (sortByAlphaAsc) { iteratees = ['reportName']; }
```

Comment:
Can we expand this into separate lines please? It's uncommon to see a single line if statement in our codebase.